flink-connector-aws/flink-connector-sqs/src/test/java/org/apache/flink/connector/sqs/sink/SqsSinkBuilderTest.java

```diff
@@ -25,14 +25,6 @@
 /** Covers construction, defaults and sanity checking of {@link SqsSinkBuilder}. */
 class SqsSinkBuilderTest {
 
-    @Test
-    void elementConverterOfSinkMustBeSetWhenBuilt() {
-        Assertions.assertThatExceptionOfType(NullPointerException.class)
-                .isThrownBy(() -> SqsSink.builder().setSqsUrl("sqlUrl").build())
-                .withMessageContaining(
-                        "No SerializationSchema was supplied to the SQS Sink builder.");
-    }
-
```

Comment:
Why did we remove this unit test?

Reply:
because we added a default SerializationSchema if it is not provided by the customer so we never end up throwing this error